en/integrations/plugin-packs/procedures/applications-databases-elasticsearch.md

```diff
@@ -1,38 +1,141 @@
+## Plugin-Pack content
 
-### Centreon Plugin
+### Monitored equipments
 
-Install this plugin on each needed poller:
+* Databases
 
-``` shell
+### Collected metrics
+
+<!--DOCUSAURUS_CODE_TABS-->
+<!--cluster-statistics-->
+
+| Metric name              | Description                              | Unit    |
+| :----------------------- | :--------------------------------------- | :---    |
+| display                  | Name of the Elasticsearch cluster        |         |
+| status                   | Status of the Elasticsearch cluster      |         |
+| nodes_total              | Number of nodes                          |         |
+| nodes_data               | Number of data                           |         |
+| nodes_coordinating       | Number of nodes coordinating             |         |
+| nodes_master             | Number of master nodes                   |         |
+| nodes_ingest             | Number of ingest data                    |         |
+| indices_count            | Number of indices                        |         |
+| shards_total             | Total number of shards                   |         |
+| shards_active            | Number of active shards                  |         |
+| shards_unassigned        | Number of unassigned shards              |         |
+| shards_relocating        | Number of relocating shards              |         |
+| active_shards_percent    | Percentage of active shards              |    %    |
+| tasks_pending            | Number of pending tasks                  |         |
+| docs_count               | Number of docs                           |         |
+| size_in_bytes            | Size of all shards assigned              |  bytes  |
+
+<!--indice-statistics-->
+
+| Metric name              | Description                              | Unit    |
+| :----------------------- | :--------------------------------------- | :------ |
+| display                  | Name of the Elasticsearch cluster        |         |
+| status                   | Status of the Elasticsearch cluster      |         |
+| shards_active            | Number of active shards                  |         |
+| shards_unassigned        | Number of unassigned shards              |         |
+| docs_count               | Number of docs                           |         |
+| size_in_bytes_primaries  | Size of all primary shards               |  bytes  |
+| size_in_bytes_total      | Total size of all shards assigned        |  bytes  |
+
+<!--license-->
+
+| Metric name        | Description                              | Unit    |
+| :----------------- | :--------------------------------------- | :------ |
+| type               | Licence type                             |         |
+| status             | Licence status                           |         |
+| issued_to          | Licence owner                            |         |
+| issue_date         | Licence issue date                       |  date   |
```

Comment:
date is not an UOM see https://www.monitoring-plugins.org/doc/guidelines.html#AEN201 paragraph

Reply:
side note: you can just remove the Unit column

Reply:
@mhuber-centreon did you fix it ?